runtime/kusama/src/lib.rs

```diff
@@ -1737,6 +1738,8 @@ sp_api::impl_runtime_apis! {
 		{
 			parachains_runtime_api_impl::validation_code_hash::<Runtime>(para_id, assumption)
 		}
+
+		fn disputes() -> Vec<(u32, CandidateHash, DisputeState)> { todo!() }
```

Comment:
is this really needed? I thought this gets generated. @tdimitrov ?

Reply:
If `disputes` are not implemented on Kusama - no, it's not needed.

However I see that the 'base version' of the runtime API is bumped to 3: https://github.com/paritytech/polkadot/pull/6161/files#diff-ea8693be4644e18e096279d95fb9277c104382a3ce7c473d388c04be9facfc9eL122

This makes the method required. I assume this todo will be replaced with a proper implementation?
```
	fn disputes() -> Vec<(SessionIndex, CandidateHash, DisputeState<BlockNumber>)> {
		runtime_parachains::runtime_api_impl::vstaging::get_session_disputes::<Runtime>()
	}
```

Reply:
@s0me0ne-unkn0wn please don't forget to fix this. It will crash the runtime otherwise.